### ISSUE TITLE
Question on readme

### ISSUE THREAD

**Issue by Westie83** [Westie83/score-bored#1]
where can the instructions be found to install the .tool-versions file?

**Comment by gcentauri** [comment]
The `.tool-versions` file _should_ be in the repo already. If it is not, thats a mistake on my part :)

ASDF also has commands which will create it, for example:

```
asdf global ruby latest
```
would create a file in your home directory like `~/.tool-versions` with the latest version of Ruby in it.

Here is the documentation from asdf-vm https://asdf-vm.com/guide/getting-started.html#_6-set-a-version

**Comment by Westie83** [comment]
i have cloned the repo and i am hoping i can now run asdf install, however, i still have some questions...

is the tool versions file hidden or is it missing? in my clone i only see:
app  config     db       Gemfile.lock  log     Rakefile   storage  tmp
bin  config.ru  Gemfile  lib           public  README.md  test     vendor

also, is "asdf install" all i have to do, and it will find the tool versions file in this repository and basically update whatever is in there? if i'm understanding the directions right, it won't update tools that aren't already installed so i would first have to verify that i have the tools, and asdf install just updates them.

**Comment by gcentauri** [comment]
> is the tool versions file hidden or is it missing?

`.tool-versions` is a hidden file. On UNIX/POSIX type systems, anything that starts with a `.` is "hidden". Generally configuration files follow this. This repo has a few, `.gitattributes` and `.gitignore` are both used by ... `git` and `.ruby-version` is the "traditional" way to mark which version of Ruby is used. I'm not sure why I have both `.ruby-version` and `.tool-versions` here.

https://github.com/Westie83/score-bored/blob/main/.tool-versions

> also, is "asdf install" all i have to do, and it will find the tool versions file in this repository and basically update whatever is in there? if i'm understanding the directions right, it won't update tools that aren't already installed so i would first have to verify that i have the tools, and asdf install just updates them.

You will have to install the asdf plugins for `asdf install` to work (i think), but I'm curious what happens if you dont. The plugins give asdf the knowledge of where to find and how to build a tool. like `asdf plugin add ruby` sets it up to make it so `asdf install ruby 2.7.8` will know how to find and build it.

`asdf install` by itself will look for a `.tool-versions` file in the directory you're working in, and if it can't find one, then in your home directory. Then if there are any things listed that ASDF hasn't installed, it will install it for you, in a special ASDF location so it won't clobber a version of a tool that is installed _without_ ASDF.

There is a lot to it, so please ask more questions if you have them.

TLDR;

`asdf install` in the repo directory will look for `.tool-versions` there and then install those versions of those tools.
